README.md

```diff
@@ -1 +1,56 @@
-# portfolio-mobile-first
\ No newline at end of file
+# https://github.com/eyoab01/portfolio-mobile-first
+This page is about my portfolio as a software developer
+
+## Built With
+HTML and CSS
+
+## Live Demo
+
+# https://github.com/eyoab01/portfolio-mobile-first
+
+
+## Setup
+download linters folder from microverse github and copy it to your project.
+
+## Install
+install the linters and the fontawesome library. (Preferred)
+
+To get a local copy up and running follow these simple steps.
+
+## Prerequisites
+Install code editor to be able to run the commands in the terminal. VSCode 
+## Setup
+ download the linters folder and copy it to your project You can clone the project through git clone https://github.com/eyoab01/portfolio-mobile-first
+
+## Install
+Install the linters folder to your project. Install the fontawesome for icons in the project 
+
+## Deployment link
+
+My portifolio website can be viewed at:  [https://github.com/eyoab01/portfolio-mobile-first]
+
+
+### Authors
+👤 Tesfaye Teklu Feyissa
+
+GitHub: @eyoab01
+Twitter: @tesfayeteklu10
+LinkedIn: www.linkedin.com/in/tesfaye-teklu-352067105
+
+### License
+This project is MIT licensed.
+
+Feel free to check the issues page.
+
+### Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+
+### Show your support
+If you like this project you can support me.
+
+### Acknowledgments
+
+I would like to thank my team mates at microverse for their support.
```

Comment:
- [ ] As suggested by the previous reviewer , Kindly I recommend you to update your README file with the new [README template](https://github.com/microverseinc/readme-template)  , **All main headers** from the [README template](https://github.com/microverseinc/readme-template) should be present in a yours repo and Please Make sure the you has added the **section Future Features**, including 1-3 potential features to be added..